tests/mkcw.bats

```diff
@@ -7,7 +7,7 @@ function mkcw_check_image() {
   # Mount the container and take a look at what it got from the image.
   run_buildah from "$imageID"
   local ctrID="$output"
-  run_buildah mount "$ctrID"
+  run_buildah --retry-delay mount "$ctrID"
```

Comment:
This really seems like the tootally wrong fix, retry is for flakes we can do nothing about. 

Looking at the errors

> Device 5d2ea557-f97e-4a36-a8e6-6996cf68449e is still in use.

and

> unmount:  ... :target is busy


do not look like network flakes, this looks like a real logic problem if buildah just isn't working.

And then even more likely buildah mount is not failing with unmount is busy AFAICT, it seems much more likely that the cryptsetup/unmount command below fail. And since they are written as plain commands without any logging you do not see which command was exectued so that needs fixing first in the test.

Overall I don't think this command belongs in this PR.